docs/pages/blog/2020-q3-update.md

```diff
@@ -0,0 +1,169 @@
+- 🛠 We have migrated parts of the codebase to TypeScript.<br />
+  We had to work on the code infrastructure of _mui-org/material-ui_ to prepare to host the date picker components that are written in TypeScript inside the lab (coming from _mui-org/material-ui-pickers_ that we will archive once we can).
+
+  <img src="/static/blog/2020-q3-update/typescript-mui.png" alt="" style="width: 299px; margin-bottom: 8px;" />
+
+  <p class="blog-description">Material-UI's repository</p>
+
+  On the other hand, we have started using TypeScript from day one on _mui-org/material-ui-x_.
+
+  <img src="/static/blog/2020-q3-update/typescript-mui-x.png" alt="" style="width: 299px; margin-bottom: 8px;" />
+
+  <p class="blog-description">Material-UI X's repository</p>
+
+- 🐙 We have migrated large parts of the test suite to react-testing-library.<br>
+  15 months ago, we introduced the very [first test](https://github.com/mui-org/material-ui/pull/15732) using the library (to replace enzyme). Last month, react-testing-library had [more downloads](https://npm-stat.com/charts.html?package=enzyme&package=%40testing-library%2Freact&from=2019-10-10&to=2020-10-10) than enzyme!
+
+  <img src="/static/blog/2020-q3-update/react-testing-library.png" alt="" style="width: 640px; margin-bottom: 40px; margin-top: 24px;" />
+
+- 💅 We have done the first iteration on the unstyled components of v5.<br />You can find a [new version](https://next.material-ui.com/components/slider-styled/#UnstyledSlider.tsx) of the slider in the lab without any styles.
+  The unstyled component weighs [5.2 kB gzipped](https://bundlephobia.com/result?p=@material-ui/lab@5.0.0-alpha.12), compared with 26 kB for the styled version when used standalone. The component is best suited for use when you want to fully customize the look without reimplementing the JavaScript and accessibility logic.<br />
+  We are also pushing in this direction to address a concern we hear from large enterprises.
+  They want to be able to go one layer down in the abstraction, to gain more control.
+
+  ```jsx
+  import SliderUnstyled from '@material-ui/lab/SliderUnstyled';
+  ```
+
+  Note that we have experimented with headless components (hooks only) in the past. For instance, you can leverage the [useAutocomplete](/components/autocomplete/#useautocomplete), and [usePagination](/components/pagination/#usepagination) hooks. However, we are pushing with unstyled first as a required step for the next item: ⬇️.
```

Comment:
A hook isn't a component, but I guess you could say that Autocomplete and Pagination are spineless components. 😁

Reply:
Well is a headless browser a bowser? :p